lua-resty-tarantool-scm-1.rockspec

```diff
@@ -0,0 +1,17 @@
+package = "lua-resty-tarantool"
+version = "scm-1"
+source = {
+   url = "git+ssh://git@github.com/darkwiiplayer/lua-resty-tarantool"
+}
+description = {
+   summary = "Tarantool integration for Openresty",
+   detailed = "Library for working with tarantool from nginx with the embedded Lua module for Openresty.",
+   homepage = "github.com/perusio/lua-resty-tarantool",
+   license = "all rights reserved"
```

Comment:
For now I set the license to "all rights reserved", since there was no license in the repository but this should obviously be a license that actually allows people to use it. The obvious choice would be MIT, which Lua itself uses.

Reply:
Yes MIT is the proper one to use.

Reply:
I'll add that one then :)